platforms/core-configuration/model-core/src/main/java/org/gradle/internal/extensibility/ConventionAwareHelper.java

```diff
@@ -166,6 +167,7 @@ private static abstract class MappedPropertyImpl implements MappedProperty {
         private boolean cache;
         private Object cachedValue;
 
+        @Nullable
         public Object getValue(org.gradle.api.plugins.Convention convention, IConventionAware conventionAwareObject) {
```

Comment:
I would prefer (to reduce work for #24767) that this be formatted as:
```suggestion
        public @Nullable Object getValue(org.gradle.api.plugins.Convention convention, IConventionAware conventionAwareObject) {
```
It matches the intention more closely and will cleanly migrate to JSpecify, unlike the current position which will need a refactor as it isn't in the right position for `TYPE_USE`.

Reply:
How should it work with cases like this, when there is no modifiers between "method annotation" and the return type?

https://github.com/gradle/gradle/blob/0d42c047f4337fbe27ea63abffc6e11aa0594dbe/subprojects/core/src/main/java/org/gradle/api/internal/DefaultNamedDomainObjectCollection.java#L556-L557

If I move the annotation next to the return type and do auto-formatting in IntelliJ, it moves `@Nullable` back on a separate line. I think whatever convention code-style we come up with, it should be supported by auto-formatting, and not reverted by it.

Reply:
Okay, I moved `@Nullable` back on a separate line for cases when there are no modifiers between it and the return type of a method.